api/protobuf-spec/mmlogic.proto

```diff
@@ -89,14 +92,21 @@ service MmLogic {
   //        the backend api along with your match results.
   // OUTPUT: a Result message with a boolean success value and an error string
   // if an error was encountered
-  rpc CreateProposal(CreateProposalRequest) returns (CreateProposalResponse) {}
+  rpc CreateProposal(CreateProposalRequest) returns (CreateProposalResponse) {
+    option (google.api.http) = {
+      put: "/v1/logic/match-proposals/{match.id}/{match.error}"
```

Comment:
This shouldn't have field parameters in the URL. Everything should go in the body. Typically creates do not have identifiers in them.

Reply:
Quoted from [gRPC HTTP Rules](https://cloud.google.com/endpoints/docs/grpc-service-config/reference/rpc/google.api#httprule):

"NOTE: the field paths in variables and in the body must not refer to repeated fields or map fields."

"The name of the request field whose value is mapped to the HTTP body, or * for mapping all fields not captured by the path pattern to the HTTP body. NOTE: the referred field must not be a repeated field and must be present at the top-level of request message type."

Would it means the repeated types are not allowed in the HTTP request body? The CreateProposalRequest messages contains a MatchObject type, which includes a repeated PlayerPool and repeated Roster. Is there any other workarounds to this limitation?